### ISSUE TITLE
Render NGP scene trained on cloud machine

### ISSUE THREAD

**Issue by alex-klimov** [NVlabs/instant-ngp#865]
We have powerful cloud machine training NGP scene. What are the options to interact with that scene from cheap laptop? We need to rotate camera and do a few clicks.

**Comment by barney2074** [comment]
have you tried saving a snapshot (on the cloud machine)
Then just loading the snapshot on the low power one ?

I'm assuming most of the GPU horsepower is in training the model, not viewing it

Andrew

**Comment by alex-klimov** [comment]
Yes, that was considered. Two things prevented us from going that direction. The first - if running trained model on another hardware is not recommended/supported approach, then it might break tomorrow even if it works today. The second - to debug our scenarios we need to place STL objects into the scene and visualize their movements. That means patching ray tracing, allowing to record movements by frame, etc. For now, we are back to extracting point cloud and mesh.
